deploy/docs/Best_Practices.md

```diff
@@ -47,14 +47,21 @@ When Multiline is On, if the first line matches `Parser_Firstline`, the rest of
 Parser_Firstline multi_line
 Parser_1 optional_parser
 ```
-### Collecting Log Lines Over 16KB
+### Collecting Log Lines Over 16KB (with multiline support)
 Docker daemon has a limit of 16KB/line so if a log line is greater than that, it might be truncated in Sumo.
```

Comment:
nitpick (I know this wasn't changed in this PR)

```suggestion
Docker daemon has a limit of 16KB/line so if a log line is longer than that, it might be truncated in Sumo.
```